src/js/view/title.js

```diff
@@ -42,7 +42,11 @@ define([
             this.description.innerHTML = (playlistItem.description && model.get('displaydescription')) ?
                 playlistItem.description : '';
 
-            this.show();
+            if(this.title.innerHTML !== '' || this.description.innerHTML !== ''){
```

Comment:
Use firstChild. it's less computational than innerHTML when it comes to checking if the element is empty:

`if(this.title.firstChild || this.description.firstChild){`